README.md

```diff
@@ -208,6 +208,33 @@ int total = new LengthOf(
 ).intValue();
 ```
 
+To get a set of unique elements by providing varargs:
```

Comment:
@fanifieiev `To get a set` sounds a little bit vague, in my opinion. Maybe it's better to start with `To create a set`?

Reply:
> @fanifieiev `To get a set` sounds a little bit vague, in my opinion. Maybe it's better to start with `To create a set`?

@iakunin Agree, will fix.